app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -1,71 +1,93 @@
 <template>
   <div>
-    <h1>{{ title }}</h1>
+    <h1 class="visualizations-contracts-show__title">
+      {{ title }}
+    </h1>
 
     <p v-if="description">
       {{ description }}
     </p>
 
-    <div class="pure-g p_2 bg-gray">
-      <div class="pure-u-1 pure-u-lg-1-2">
-        <label class="soft">{{ labelAsignee }}</label>
-        <div class="">
-          <router-link
-            id="assignee_show_link"
-            :to="{ name: 'assignees_show', params: {id: assignee_routing_id } }"
-          >
-            <strong class="d_block">{{ assignee }}</strong>
-          </router-link>
-          <span v-if="assignee_id">
-            {{ assignee_id }}
-          </span>
+    <div class="pure-g p_2 bg-gray visualizations-contracts-show">
+      <ContractsShowLabelHeader
+        class="visualizations-contracts-show__block"
+        :label="labelAwardingEntity"
+        :value="contractor"
+        :icon="'building'"
+      />
+      <ContractsShowLabelHeader
+        :label="labelStatus"
+        :value="status"
+        :icon="'columns'"
+      />
+      <ContractsShowLabelHeader
+        :label="labelType"
+        :value="contract_type"
+        :icon="'clipboard-list'"
+      />
+      <ContractsShowLabelHeader
+        :label="labelProcess"
+        :value="process_type"
+        :icon="'archive'"
+      />
+      <ContractsShowLabelHeader
+        class="visualizations-contracts-show__block"
+        :label="labelCategory"
+        :value="category_title"
+        :icon="'tag'"
+      />
+      <div class="visualizations-contracts-show__body">
+        <div
+          v-show="isNotAMinorContract"
+          class="pure-u-1 pure-u-lg-1-2"
+        >
+          <div class="pure-u-1 pure-u-lg-1-1 visualizations-contracts-show__body__group">
+            <span class="visualizations-contracts-show__text__header">{{ labelTender }}</span>
+            <span class="visualizations-contracts-show__text">{{ parseDate(open_proposals_date) }}</span>
+            <i
+              v-show="showArrowDate"
+              class="fas fa-arrow-right"
+            />
+            <span class="visualizations-contracts-show__text">{{ parseDate(submission_date) }}</span>
+          </div>
+          <ContractsShowLabelGroup
+            :label="labelBidDescription"
+            :value="initial_amount_no_taxes | money"
+          />
+          <ContractsShowLabelGroup
```

Comment:
The number of proposals now comes within the data, in a column named number_of_proposals

Reply:
Done